Automattic/jetpack
# Jetpack | Fix recommendations CTA being marked as external

## Proposed changes:

* Updates Jetpack recommendations section to update Social CTA and description

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to `wp-admin/admin.php?page=jetpack#/recommendations/publicize`
* Confirm that the CTA says, "View Jetpack Social settings" and links to the social settings section
* Goto `/wp-admin/admin.php?page=jetpack#/recommendations/unlimited-sharing-activated`
* Confirm that the description is same as the above section and the CTA points to social settings section


<img width="647" alt="image" src="https://github.com/Automattic/jetpack/assets/18226415/40b7bab9-4e34-47ba-ab84-48569feb4caa">
